.github/workflows/sync.yml

```diff
@@ -0,0 +1,45 @@
+name: Sync Dependencies
+on:
+  pull_request:
+    branches:
+      - 'main'
+    types:
+      - 'opened'
+      - 'synchronize'
+      - 'reopened'
+jobs:
+  sync-dependencies:
+    if: startsWith(github.actor, 'dependabot')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
```

Comment:
@LBishop234  Yes but because of how the pipeline's environment is configured I needed to specify this otherwise the commit would not be pushed, the pipeline initially failed without this. I could have shown you the pipeline failure but I cleaned up the logs and deleted it yesterday annoyingly.